assets/base.css

```diff
@@ -2518,3 +2518,46 @@ details-disclosure > details {
   border-left: none;
   border-right: none;
 }
+
+@supports not (inset: 10px) {
```

Comment:
The issue is that we do use `inset` in our codebase. By adding this, we're in a way contradicting ourselves with regards to what browser support we are offering? We should either remove instances of `inset` in the theme or use a different approach here.

Reply:
I think here because it breaks the whole layout it's a bit more problematic maybe. I can see that it's then opening a door to more questioning about other supported/unsupported things we use and when to add support and when not to worry about it. 
People can still use the theme and go on about their purchase, so maybe we shouldn't worry about it too much 🤷 

cc: @tyleralsbury